data/campaigns/tutorial03_seafaring.wmf/scripting/texts.lua

```diff
@@ -164,32 +178,52 @@ expedition3 = {
    )
 }
 
-conclusion = {
+expedition4 = {
    position = "topright",
    field = port_on_island,
    title = _"Conclusion",
    body = (
       h1(_"Congratulations") ..
```

Comment:
This was originally the final message of the tutorial. I don't think the titles `_"Conclusion"` and `_"Congratulations"` fit any more. How abut "Colony" / "Your colony has been founded" / "New colony" or something like that?